### PR TITLE
14639: 1995: Update button and question content

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As a developer, I need to update the 1995 so that it is up to date with the current DEPO patterns and practices.

[Originating Issue](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/14639)

## Screenshots
Screen Shot 2020-10-15 at 3 53 37 PM
Screen Shot 2020-10-15 at 3 54 56 PM

## Acceptance criteria
- [x] In chapter 3, the "Add Another Service Period" button is replaced with "Add another service period"
- [x] In chapter 5, the "Do you have a parent who is dependent on your financial support?" question is replaced with "Do you have a parent who depends on you for financial support?"

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by zurbergram — APPROVED** [review]
What other forms does this affect?

**Comment by elder-joseph** [comment]
The platform specific casing change will affect the following edu-benefits forms:
- 1995
- 1990
- 1990E
- 1990N
- 5490
- 5495

**Review by sonntag-adam — APPROVED** [review]

File: src/platform/forms-system/test/js/review/ArrayField.unit.spec.jsx [hunk 1]
```diff
@@ -155,7 +155,7 @@ describe('Schemaform review <ArrayField>', () => {
     tree.getMountedInstance().handleAdd();
     expect(tree.everySubTree('h5')[0].text()).to.equal('New Item name');
     expect(tree.everySubTree('button')[2].text()).to.equal(
-      'Add Another Item name',
+      'Add another Item name',
```
Comment (rianfowler) [inline src/platform/forms-system/test/js/review/ArrayField.unit.spec.jsx:158]: "Item" is capitalized mid-sentence.

**Review by rianfowler — CHANGES_REQUESTED** [review]
I think there was a decision related to this kind of change. @department-of-veterans-affairs/vsp-design-system-fe would know more.

**Review by bkjohnson — COMMENTED** [review]
I think this is fine, it follows the relevant practices/patterns:

- https://design.va.gov/content-style-guide/capitalization

However, I will try to find more info about that decision that Rian mentioned, because I also remember something along those lines.

**Review by rianfowler — APPROVED** [review]

**Review by bkjohnson — APPROVED** [review]
This issue captures the decision, along with a PR that was working towards removing the "Add Another" casing pattern:

https://github.com/department-of-veterans-affairs/va.gov-team/issues/13684

The original PR that made the shift to sentence case for the "Add Another" pattern, while still allowing teams the option of opting out: https://github.com/department-of-veterans-affairs/vets-website/pull/14061

---

This more global approach is fine for now (it satisfies the first Acceptance Criteria in the issue), and if we decide that we want to meet those second and third AC, that is something that the design system team will do.

**Comment by elder-joseph** [comment]
@bkjohnson Thank you for providing the context for this issue and the decision made around it.

**Pull request merged**